code/ATMOSPHERICS/components/unary/cold_sink.dm

```diff
@@ -25,27 +25,27 @@
 	..()
 
 /obj/machinery/atmospherics/unary/freezer/atmos_init()
-	if(node)
+	if(node1)
```

Comment:
the vast majority of the changes are just swapping node for node1 on unary machines. far fewer files than it appears have any meaningful change